server/api/user.py

```diff
@@ -159,7 +164,7 @@ def inviteUser(self, params):
     self.requireParams(['login', 'email', 'firstName', 'lastName'], params)
     if 'validityPeriod' in params:
         try:
-            validityPeriod = params['validityPeriod']
+            validityPeriod = float(params['validityPeriod'])
```

Comment:
The `float` built-in function strips whitespace itself, so I don't think it's necessary.